packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.9.17+4
 
+* Adds possibility to use any supported FPS and fixes crash when using unsupported FPS.
```

Comment:
Where is the new test that triggers the codepath that used to crash?

Reply:
When the real `captureDevice.activeVideoMinFrameDuration` (or ...Max...) is set to value outside of what is supported by active format it immediately throws an objc exception. For tests, mocked wrappers are used so this cannot be triggered. But the test `testSettings_ShouldSelectFormatWhichSupports60FPS` tries to set 60 fps which actually triggered this crash on my device for media presets other than 1280x720.